src/reaper_osara.cpp

```diff
@@ -272,6 +272,20 @@ const char* getFolderCompacting(MediaTrack* track) {
 	return ""; // Should never happen.
 }
 
+string formatTrackWithName(MediaTrack* track) {
```

Comment:
I actually think it was related to it for the surface to be able to report muted tracks, however since that turned out to be unreliable, I removed that and decided that splitting this code out was a good thing. I can revert it if you desire.